cogs5e/gametrack.py

```diff
@@ -434,15 +434,25 @@ async def spellbook_remove(self, ctx, *, spell_name):
     async def customcounter(self, ctx, name=None, *, modifier=None):
         """Commands to implement custom counters.
         When called on its own, if modifier is supplied, increases the counter *name* by *modifier*.
-        If modifier is not supplied, prints the value and metadata of the counter *name*."""
+        If modifier is not supplied, prints the value and metadata of the counter *name*.
+        """
         if name is None:
             return await self.customcounter_summary(ctx)
         character: Character = await Character.from_ctx(ctx)
         counter = await character.select_consumable(ctx, name)
 
+        cc_embed_title = counter.title if counter.title is not None else counter.name
+
+        # replace [name] in title
+        cc_embed_title = cc_embed_title.replace('[name]', character.name)
+
+        cc_desc = counter.desc if counter.desc else ''
```

Comment:
The use of `cc_desc` below can probably be changed to just checks on `counter.desc`, and this variable removed